src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -7677,15 +7810,16 @@ public void TestNullCheckedArgWithSpaceAfterBangs()
                     N(SyntaxKind.CloseBraceToken);
                 }
             }
+            EOF();
         }
 
         [Fact]
         public void TestNullCheckedArgWithSpaceBeforeBangs()
         {
             UsingStatement(@"void M(string name ! !=null) { }", options: TestOptions.RegularPreview,
-                    // (1,22): error CS1525: Invalid expression term '!'
+                    // (1,20): error CS1003: Syntax error, '!!' expected
```

Comment:
>'!!' expected [](http://example.com/codeflow?start=59&length=13)

Are we testing this case with trailing and leading trivia on the two tokens? Perhaps something like:

```csharp
void M(string name ! /*comment1*/
/*comment2*/!);
```